src/ipwndfu/device_platform.py

```diff
@@ -116,11 +116,11 @@ def __init__(
 
     def name(self):
         if 0x8720 <= self.cpid <= 0x8960:
-            return f"s5l{self.cpid}xsi"
+            return f"s5l{hex(self.cpid)[2:]}xsi"
```

Comment:
i think the canonical way to do this is with format specifiers : `f"something with {input:02X}"`

Reply:
`hex()` is wrong here anyway because it adds a `0x` prefix.